src/translation/translation_nb_NO.ts

```diff
@@ -26,7 +26,7 @@
     <message>
         <location filename="../util.cpp" line="597"/>
         <source>(build)</source>
-        <translation type="unfinished"></translation>
+        <translation>(bygg)</translation>
```

Comment:
Probably worth reverting this for now, as I think it's "build" rather than "compile"...